### PR TITLE
many: install snaps in devmode on distributions without complete apparmor and seccomp support

### PR DESCRIPTION
**Pull request by zyga**
This branch changes the release module to add APIs for detecting distributions that don't yet support all of the apparmor and seccomp features that constitute the security system. On distributions like that the daemon will now unconditionally set the devmode flag on install and sideload requests.

### CODE REVIEW

**Comment by chipaca** [comment]
:+1:

File: release/release.go [hunk 1]
```diff
@@ -38,6 +38,21 @@ type OS struct {
 	Codename string
 }
 
+// IsDevModeDistro returns true if the distribution doesn't implement required
+// security features for confinement.
+func (os *OS) IsDevModeDistro() bool {
```
Comment (niemeyer) [inline release/release.go]: `ForceDevMode` sounds like a better name for this method.

**Comment by niemeyer** [comment]
LGTM

File: release/release.go (hunk 1 above)
Comment (zyga) [inline release/release.go]: Updated

**Pull request merged**